mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -159,11 +162,11 @@ public void setInMailboxes(MessageId messageId, List<MailboxId> mailboxIds, Mail
         MessageIdMapper messageIdMapper = mailboxSessionMapperFactory.getMessageIdMapper(mailboxSession);
         MailboxMapper mailboxMapper = mailboxSessionMapperFactory.getMailboxMapper(mailboxSession);
 
-        allowOnMailboxSession(mailboxIds, mailboxSession, mailboxMapper);
-
-        List<MailboxMessage> mailboxMessages = messageIdMapper.find(ImmutableList.of(messageId), MessageMapper.FetchType.Full)
+        assertRightsOnMailboxes(mailboxIds, mailboxSession, Right.Read);
```

Comment:
I can't understand that right check, can anybody enlight me ?

Reply:
maybe that should be Lookup instead

Reply:
I introduced it for not breaking previous tests. I could very well remove it.